docs/context.asciidoc

```diff
@@ -0,0 +1,16 @@
+[float]
+==== Context
+An event's context bundles information regarding the environment in which it is recorded.
+It describes the `service` in which the event is captured, 
+the `system` in which the monitored service is running and the event's `process` information.
+
+It can also contain information about the authenticated `user`. 
+
+An event's context can also include information about the request leading to the event and the response of processing the event. 
+In case a http request is captured, it contains information about the `url`, `cookies`, `body`, `headers`, etc.
+
+The agents provide some configuration options with which the users can also capture customized information.
+The non-indexed information is captured within a `custom` object,
+while the indexed information is stored within `tags`.
```

Comment:
are we good to assume users know what indexed means?  maybe searchable instead ?